### PR TITLE
Fix dependency issue with SNB

### PR DESCRIPTION
**Pull request by aelred**
Also take the opportunity to simplify some of the horrible scripts

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -104,6 +103,21 @@ def buildGrakn() {
...
+        .--.     ___/     \
+       /    `.-""   `-,    ;
+      ;     /     O O  \  /
+      `.    \          /-'
+     _  J-.__;      _.'
+    (" /      `.   -=:
+     `:         `, -=|   ______________________________
+      |  F\    i, ; -|  /                              \
+      |  | |   ||  \_J <  DON'T FORGET TO REMOVE THIS! |
+      mmm! `mmM Mmm'    \______________________________/
```
Comment (fppt) [inline Jenkinsfile]: I keep seeing elephants!!!

File: grakn-test/test-snb/pom.xml [hunk 2]
```diff
@@ -29,27 +24,45 @@
...
-                    </descriptorRefs>
-                </configuration>
-            </plugin>
-        </plugins>
-    </build>
+        <!--
+            This dependency is duplicated from the main pom deliberately.
+            This is because com.ldbc.driver:jeeves provides its own (older) Guava dependency. This is included directly
+            in their distribution, not as a proper dependency so we can't actually exclude it. Instead, we make sure
+            that OUR guava dependency is loaded FIRST by putting it before the com.ldbc.driver:jeeves dependency.
```
Comment (fppt) [inline grakn-test/test-snb/pom.xml]: Hacky. . . but I am not sure of another way to handle this.

**Comment by aelred** [comment]
@fppt I scared the elephant away. please re-review!

**Comment by fppt** [comment]
I am no longer scared of the elephant affecting this merge.

**Pull request merged**